source/state_representation/include/state_representation/StateType.hpp

```diff
@@ -29,6 +29,8 @@ enum class StateType {
   GEOMETRY_SHAPE,
   GEOMETRY_ELLIPSOID,
   TRAJECTORY,
+  ANALOG_IO_STATE,
+  DIGITAL_IO_STATE,
```

Comment:
the most minor of nitpicks but I would prefer `digital` to come before `analog`. It's a little like the elevation of `bool` to `double` in the order of parameter types, but also more common to how you would say "digital and analog IOs"

Reply:
you marked this as resolved but if it doesn't cause any other issues I would still prefer it the other way around

Reply:
Oups sorry, I think I did on the proto branch instead.